crates/re_viewport/src/viewport.rs

```diff
@@ -770,7 +770,7 @@ impl<'a, 'b> egui_tiles::Behavior<SpaceViewId> for TabViewer<'a, 'b> {
 
     // Callbacks:
 
-    fn on_edit(&mut self) {
+    fn on_edit(&mut self, _action: egui_tiles::EditAction) {
```

Comment:
Seemed like the right call, but someone more familiar with the matter might want to check

Reply:
well nevermind this has already changed on `main` :laughing: